src/Conversations/Conversation.php

```diff
@@ -123,10 +125,10 @@ public function terminate(Nutgram $bot, ?int $userId = null, ?int $chatId = null
     }
 
     /**
-     * @param  bool  $skipHandlers
+     * @param  bool|UpdateType[]  $skipHandlers
      * @return Conversation
      */
-    protected function setSkipHandlers(bool $skipHandlers): self
+    protected function setSkipHandlers(mixed $skipHandlers): self
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏same here, array|bool

Reply:
This is a BC for those who have overridden this property.